modules/apps/saved-content/saved-content-web/src/main/java/com/liferay/saved/content/web/internal/portal/profile/ModulePortalProfile.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.saved.content.web.internal.portal.profile;
+
+import com.liferay.portal.profile.BaseDSModulePortalProfile;
+import com.liferay.portal.profile.PortalProfile;
+import com.liferay.saved.content.web.internal.portlet.MySavedContentPortlet;
+
+import java.util.Arrays;
+
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Alicia García
+ */
+@Component(service = PortalProfile.class)
+public class ModulePortalProfile extends BaseDSModulePortalProfile {
+
+	@Activate
+	protected void activate(ComponentContext componentContext) {
+		init(
+			componentContext,
+			Arrays.asList(
+				PortalProfile.PORTAL_PROFILE_NAME_CE,
+				PortalProfile.PORTAL_PROFILE_NAME_DXP),
+			MySavedContentPortlet.class.getName());
+	}
+
```

Comment:
(question) Why do we need this? It is the first time I've seen this in a module :thinking: